Source/Core/Core/AchievementManager.cpp

```diff
@@ -487,7 +494,7 @@ void AchievementManager::CloseGame()
       m_queue.Cancel();
       m_image_queue.Cancel();
       rc_client_unload_game(m_client);
-      m_system = nullptr;
+      m_system.store(nullptr, std::memory_order_acquire);
```

Comment:
Did you mean to use `memory_order_acquire` here? I'm rather unsure what a store with `memory_order_acquire` even does.

Reply:
Oops. As far as I know, an acquire store basically acts like a relaxed store.